pages/community/precice-workshop-2022.md

```diff
@@ -19,12 +19,196 @@ If you have any questions regarding the workshop, please contact us at [`precice
+    analyzed.</p>
+    </details>
+
+    <details class="workshop-event" id="Sircar"><summary>Arpan Sircar: Volumetric coupling of OpenFOAM for multi‐physics simulations of fusion reactor blankets</summary>
+    <p>Authors: <a>Arpan Sircar</a><br/></p>
+    <p>preCICE was mainly developed for surface coupling of multiple codes. In the present work, the
+    capability of volumetric coupling using preCICE’s OpenFOAM adapter is tested. This is
+    performed using cell‐level interpolations and operator‐splitting of OpenFOAM solvers. The
+    sensitivity to different mapping schemes, time interpolation windows and acceleration
+    techniques are studied. This is primarily a work in progress to ascertain whether preCICE’s
+    capabilities can be used out of the box to perform volumetric coupling. While current tests are
+    only limited to OpenFOAM solver, coupling with other home‐grown and commercial codes
+    such as Diablo for structural simulations and MCNP/Shift for neutronic simulations are
+    underway.</p>
+    <p>The broader goal of the current project is to develop an integrated simulation environment for
+    the coupled neutronic, thermal‐hydraulic, and plasma‐physics simulation of plasma, vacuum
+    vessel, and blanket of conceptual fusion reactors.</p>
+    </details>
+
+* 15:00 - 16:30 preCICE Course II (Main Hall)
+
+  <details class="workshop-event" id="courseII"><summary>preCICE Course II: Implicit coupling for Conjugate Heat Transfer</summary>
+  <p>Instructors: <a href="https://www.in.tum.de/en/i05/people/personen/gerasimos-chourdakis/">Gerasimos Chourdakis</a>, <a href="https://www.simtech.uni-stuttgart.de/exc/people/Uekermann/">Benjamin Uekermann</a><br/>
+  Affiliation: Technical University of Munich, University of Stuttgart, preCICE developers.</p>
+  <p>A hands-on introduction to implicit coupling details in preCICE, recommended for new users that want to learn how to make their coupled simulations more accurate and numerically efficient.</p>
+  <p>We will couple OpenFOAM and Nutils for Conjugate Heat Transfer, discussing the basic methods of the preCICE API and the structure of the configuration file. We will then also look into tools useful for developing and debugging coupled simulations.</p>
+  <p>We will need preCICE v2.3, Python 3.6 or newer, the Python bindings, Nutils 6, OpenFOAM (e.g. v2112), the latest OpenFOAM adapter, and ParaView. If you prefer to try everything in a Virtual Machine, you can use the <a href="https://app.vagrantup.com/precice/boxes/precice-vm">Vagrant Box</a> (VirtualBox image) <a href="https://app.vagrantup.com/precice/boxes/precice-vm">precice/precice-vm</a>.</p>
+  <p>If you are new to preCICE, see "preCICE Course I" first. If you already understand the basics of an adapter, you can directly start here.</p>
+  </details>
+
+* 16:30 - 17:00 Break (Offline)
+* 17:25 - 18:15 Networking/Social Event (Main Hall)<br/>
```

Comment:
We have two options:
* Pay for two hours
* Pay for a day

If we pay for a day then we can use Gather.Town that whole day and also have an open-ended social event. I am tending towards paying for the day because some people might want to hang around for a longer time. Opinions?

Reply:
Lets have the social event for two hours for now.